lib/Runtime/Library/ArrayBuffer.cpp

```diff
@@ -707,7 +713,12 @@ namespace Js
         {
             freeFunction(this->buffer);
             this->buffer = nullptr;
-            this->recycler->ReportExternalMemoryFree(this->bufferLength);
+
+            // for projection array buffers we have already reported freeing on detach
+            if (this->allocationType != ArrayBufferAllocationType::CoTask)
```

Comment:
@Cellule - I had to do a small change here. Comparing template functions would not build in Core, so I changed this to compare `allocationType`

Reply:
no longer applicable

---
In reply to: [186249460](https://github.com/Microsoft/ChakraCore/pull/5099#discussion_r186249460) [](ancestors = 186249460)